Apps/Sandcastle/gallery/Image-Based Lighting.html

```diff
@@ -107,15 +107,19 @@
 
     subscribeParameter('luminanceAtZenith');
 
-    Sandcastle.addToggleButton('Use procedural image', false, function(checked) {
-        if (!checked) {
-            model.sphericalHarmonicCoefficients = coefficients;
-            model.specularEnvironmentMaps = environmentMapURL;
-        } else {
-            model.sphericalHarmonicCoefficients = undefined;
-            model.specularEnvironmentMaps = undefined;
-        }
-    });
+    if (Cesium.OctahedralProjectedCubeMap.isSupported(viewer.scene.context)) {
+        Sandcastle.addToggleButton('Use procedural environment lighting', false, function(checked) {
+            if (!checked) {
+                model.sphericalHarmonicCoefficients = coefficients;
+                model.specularEnvironmentMaps = environmentMapURL;
+            } else {
+                model.sphericalHarmonicCoefficients = undefined;
+                model.specularEnvironmentMaps = undefined;
+            }
+        });
+    } else {
+        console.log("This browser doesn't support texture environment maps.");
```

Comment:
```suggestion
        console.log('This browser doesn't support specular environment maps.');
```

Use single-quotes for strings. Also `texture environment maps` is a bit redundant so I replaced `texture` with `specular` which is what the option is called.